### ISSUE TITLE
File not found exception

### ISSUE THREAD

**Issue by VinayPalaksha1** [xceedsoftware/DocX#240]
I'm getting file not found exception when I tried to use DocX to read a word document.

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Xceed.Words.NET;
using System.IO;

namespace ConsoleApp4
{
    class Program
    {
        static void Main(string[] args)
        {
            DocX doc = DocX.Load(@"C:\Users\Vinay\Downloads\Context related help");
            var head = doc.Paragraphs.Where(i => i.StyleName.Equals("Title", StringComparison.InvariantCultureIgnoreCase));

            var paralist = doc.Paragraphs;

            var firstPara = paralist.First();
            string text = firstPara.Text;

            var titleparagrap = doc.Paragraphs.Where(i => i.StyleName.Equals("Title", StringComparison.InvariantCultureIgnoreCase));
            Console.WriteLine(titleparagrap);
        }
    }
}

**Comment by PrzemyslawKlys** [comment]
`DocX doc = DocX.Load(@"C:\Users\Vinay\Downloads\Context related help"); `

Doesn't seem like a correct file path.

**Comment by VinayPalaksha1** [comment]
May I know whats wrong with path, there are no build errors nor red lines. When I run ,i get exception

**Comment by PrzemyslawKlys** [comment]
There is no file at the end? You want to Load file? What file? there is no ThisIsMyWordFile.docx ?

**Comment by VinayPalaksha1** [comment]
Sorry that was a typo. 

DocX doc = DocX.Load(@"C:\Users\Vinay\Downloads\Context related help.docx");

even with this, Im getting exception!

**Comment by PrzemyslawKlys** [comment]
Does the file exists? DocX.Load is for Loading existing file. If you want to create new file you don't use DocX.Load. I suggest ... go thru examples.

**Comment by VinayPalaksha1** [comment]
Yes. file does exist. I just want to read an existing file. Not creating new one

**Comment by PrzemyslawKlys** [comment]
File not found exception most likely means what it means. Try to verify (code wise) that the file really is there. var file = 'path to my file' and then check for existance of that file. Maybe there are permission problems.

**Comment by VinayPalaksha1** [comment]
will check. Thanks for your help

**Comment by XceedBoucherS** [comment]
Hi,
DocX is currently not working/tested on .net Standard/.net core. 
Based on https://github.com/xceedsoftware/DocX/issues/226.
